skeleton/docs/Makefile

```diff
@@ -0,0 +1,47 @@
+##
+## Makefile for iohk-skeleton example, based on:
```

Comment:
Cute - I wrote a [latex builder](https://github.com/input-output-hk/plutus/blob/master/nix/latex.nix#L4) for plutus that does a bunch of the same things, which has the advantage that you don't have to copy the makefile, but the disadvantage that you have to manually run the appropriate `latexmk` commands yourself during development. I think this is probably nicer for people writing the docs too.

Reply:
I copied this stuff from cardano-ledger-specs.
Maybe we could also copy your latex builder to iohk-nix.